### PR TITLE
Fix JS whitespace CS

### PR DESCRIPTION
**Pull request by mvorisek**
and address https://github.com/roundcube/roundcubemail/commit/740267fac06e44aa2b12d2184cd6c2df3bea09fc#commitcomment-137942580

### CODE REVIEW

**Review by alecpl — CHANGES_REQUESTED** [review]

File: plugins/acl/acl.js [hunk 1]
```diff
@@ -42,33 +41,28 @@ if (window.rcmail) {
...
 
 // Display ACL edit form
-rcube_webmail.prototype.acl_edit = function()
-{
+rcube_webmail.prototype.acl_edit = function () {
     // @TODO: multi-row edition
     var id = this.acl_list.get_single_selection();
-    if (id)
-        this.acl_init_form(id);
+    if (id) { this.acl_init_form(id); }
```
[the remark this answers is not in the document]
Comment (mvorisek) [reply plugins/acl/acl.js:57]: I say it, but me neither - I always aim to be as much as close to the used PHP CS. I will work on this.

File: plugins/acl/acl.js [hunk 2]
```diff
@@ -142,75 +129,65 @@ rcube_webmail.prototype.acl_update = function(o)
...
 // ACL table keypress handler
-rcube_webmail.prototype.acl_list_keypress = function(list)
-{
-    if (list.key_pressed == list.ENTER_KEY)
-        this.command('acl-edit');
-    else if (list.key_pressed == list.DELETE_KEY || list.key_pressed == list.BACKSPACE_KEY)
-        if (!this.acl_form || !this.acl_form.is(':visible'))
-            this.command('acl-delete');
+rcube_webmail.prototype.acl_list_keypress = function (list) {
+    if (list.key_pressed == list.ENTER_KEY) { this.command('acl-edit'); } else if (list.key_pressed == list.DELETE_KEY || list.key_pressed == list.BACKSPACE_KEY) {
```
[the remark this answers is not in the document]
Comment (mvorisek) [reply plugins/acl/acl.js:170]: Why do you think so? The l170 is rendered in GH diff on two lines, but it is one line.

[the remark this answers is not in the document]
Comment (mvorisek) [reply plugins/acl/acl.js]: That is https://github.com/roundcube/roundcubemail/pull/9327#discussion_r1469930337 issue.

File: plugins/acl/acl.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (mvorisek) [reply plugins/acl/acl.js]: Fixed, for some reasons, AirBnB changed the default config in https://github.com/airbnb/javascript/blob/eslint-config-airbnb-v19.0.4/packages/eslint-config-airbnb-base/rules/style.js#L21.

File: plugins/help/help.js [hunk 3]
```diff
@@ -17,27 +17,28 @@
 
 // hook into switch-task event to open the help window
 if (window.rcmail) {
-    rcmail.addEventListener('beforeswitch-task', function(prop) {
+    rcmail.addEventListener('beforeswitch-task', function (prop) {
         // catch clicks to help task button
         if (prop == 'help') {
-            if (rcmail.task == 'help')  // we're already there
+            if (rcmail.task == 'help') // we're already there
+            {
```
Comment (alecpl) [inline plugins/help/help.js:24]: Can you fix this one?

File: program/js/app.js [hunk 4]
```diff
@@ -256,29 +268,47 @@ function rcube_webmail()
...
-                        .addEventListener('initrow', function(o) { ref.init_message_row(o); })
-                        .addEventListener('dblclick', function(o) { ref.msglist_dbl_click(o); })
-                        .addEventListener('keypress', function(o) { ref.msglist_keypress(o); })
-                        .addEventListener('select', function(o) { ref.msglist_select(o); })
-                        .addEventListener('dragstart', function(o) { ref.drag_start(o); })
-                        .addEventListener('dragmove', function(e) { ref.drag_move(e); })
-                        .addEventListener('dragend', function(e) { ref.drag_end(e); })
-                        .addEventListener('expandcollapse', function(o) { ref.msglist_expand(o); })
-                        .addEventListener('column_replace', function(o) { ref.msglist_set_coltypes(o); })
+                        .addEventListener('initrow', function (o) {
```
Comment (alecpl) [inline program/js/app.js:281]: This compact version was quite convenient, but I suppose it does not matter that much.

File: plugins/help/help.js (hunk 3 above)
Comment (mvorisek) [reply plugins/help/help.js]: Yes, there is no special config for `if` (https://github.com/roundcube/roundcubemail/pull/9327#discussion_r1469930337) and for anonymous functions. The change is consistent with PHP CS Fixer.

File: program/js/app.js (hunk 4 above)
Comment (mvorisek) [reply program/js/app.js:281]: Yes, there is no special config for `if` (https://github.com/roundcube/roundcubemail/pull/9327#discussion_r1469930337) and for anonymous functions. The change is consistent with PHP CS Fixer.

File: plugins/help/help.js (hunk 3 above)
Comment (mvorisek) [reply plugins/help/help.js]: Fixed. We cannot fully enable `brace-style` yet, thus this was not autofixed/detected.

**Pull request merged**